en/docs/install-and-setup/setup/deployment/deploying-icp-with-helm.md

```diff
@@ -0,0 +1,144 @@
+# Deploy the Integration Control Plane (ICP) on Kubernetes using Helm resources
+
+Follow the instructions below to deploy the ICP on Kubernetes (K8s) using Helm resources.
+
+## Before you begin
+    
+- Ensure you have an active [WSO2 Subscription](https://wso2.com/subscription). If you don't have a subscription, sign up for a [WSO2 Free Trial Subscription](https://wso2.com/free-trial-subscription).
+
+    !!! Note
+        You need an active subscription to use the updated Docker images of the ICP with your Helm resources. Otherwise, you can use the community version of the Docker images, which do not include product updates.
+    
+- Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://helm.sh/docs/intro/install/), and the [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/).
+    
+- Set up a [Kubernetes cluster](https://kubernetes.io/docs/setup/#learning-environment).
+    
+- Install the [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). 
+
+    !!! Note
+        Helm resources for WSO2 product deployment patterns are compatible with the ingress-nginx [`controller-v1.11.2`](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.11.2) release.
+
+## Step 1 - Get the Helm resources
+
+Check out the Helm resources for the WSO2 Micro Integrator Git repository.
+
+1. Open a terminal and navigate to the location where you want to save the local copy.
+2. Clone the Micro Integrator Git repository with Helm resources:
+
+    ```bash
+    git clone https://github.com/wso2/helm-mi.git
+    ```
+
+This creates a local copy of the [`wso2/helm-mi`](https://github.com/wso2/helm-mi/) repository, which includes all the Helm resources for WSO2 Micro Integrator.
+
+Let's refer to the root folder of the local copy as `<HELM_HOME>`.
+
+## Step 2 - Update the deployment configurations 
+
+Follow the steps below to configure your Micro Integrator deployment.
```

Comment:
Shouldn't this be ICP?

Reply:
Also let's mention to checkout to 4.3.x branch

Reply:
Added by https://github.com/wso2/docs-mi/pull/1530/commits/1c7ee73efc8e83197c8be971e61526ef307447cd